### PR TITLE
Updating year to 2019 in the copyright for appdata.xml

### PR DESCRIPTION
**Pull request by BunnyApocalypse**
Fixes #

### Changes Summary

- Changes year from 2018 to 2019 in the Appdata.xml section

This pull request is ready for review.

Sorry about the reverts etc, forgot to branch for each change initially.

### CODE REVIEW

**Comment by BunnyApocalypse** [comment]
A better way to do this might just be to run a find and replace on appropriate parts of the website.

File: docs/code/getting-started.md [hunk 1]
```diff
@@ -523,7 +523,7 @@ Now it's time to create the rules that will allow your app to be built as a .deb
 
           * Initial Release.
 
-         -- Your Name <you@emailaddress.com>  Friday, 20 Apr 2018 04:53:39 -0500
+         -- Your Name <you@emailaddress.com>  Friday, 20 Apr 2019 04:53:39 -0500
```
Comment (btkostner) [inline docs/code/getting-started.md]: This date is in the future...

Comment (BunnyApocalypse) [reply docs/code/getting-started.md]: I suppose this date is kind of impossible to get right seeing as its more specific than just the year, but I guess changing it to today for now wont hurt.

```suggestion
         -- Your Name <you@emailaddress.com>  Monday, 8 Apr 2019 04:53:39 -0500
```

Comment (btkostner) [reply docs/code/getting-started.md]: @bunnyapocalypse, you will need to merge that suggestion. I don't have permission to edit your branch

**Comment by btkostner** [comment]
@BunnyApocalypse Thank you for the PR!

**Pull request merged**